nf_core/pipeline-template/nextflow.config

```diff
@@ -19,7 +19,7 @@ params {
 
   // Boilerplate options
   multiqc_config = false
-  email = false
+  email = ""
```

Comment:
```suggestion
  email = null
```

Reply:
See https://github.com/nf-core/tools/issues/992

Reply:
I have also fixed all of this for the incoming PR to update the DSL2 pipeline template in https://github.com/nf-core/tools/pull/1094 (see [here](https://github.com/drpatelh/tools/blob/dsl2/nf_core/pipeline-template/nextflow.config))

Reply:
Very good point, I was not aware of that at all.
I keep my current empty version in Sarek as null is not working with current tools.

Reply:
I think it may be because you need the latest `lib/NfcoreSchema.groovy` file.

Reply:
You could use the latest files in [here](https://github.com/drpatelh/tools/tree/dsl2/nf_core/pipeline-template/lib) but you may need to update a few more things in the DSL2 pipeline template to get them to work.

Reply:
I think it was working for DSL1 at some point with `null` values 🤔 Dunno, losing track of these changes for DSL1 vs DSL2. I do know that it is working with rnaseq and viralrecon because I have the same value initialised as `null` already there as you can see [here](https://github.com/nf-core/rnaseq/blob/0fcbb0ac491ecb8a80ef879c4f3dad5f869021f9/nextflow.config#L84)

Reply:
Pretty certain that `null` works fine in DSL1 in principle 👍🏻